sdcm/cluster.py

```diff
@@ -3726,23 +3718,34 @@ def node_setup(_node: BaseNode):
             except Exception:  # pylint: disable=broad-except
                 LOGGER.warning("Failure settings shards for node %s in Argus.", _node)
                 LOGGER.debug("Exception details:\n", exc_info=True)
-            _queue.put((_node, exception_details))
-            _queue.task_done()
+            task_queue.put((_node, exception_details))
+            task_queue.task_done()
+
+        @raise_event_on_failure
```

Comment:
hard to get failure if we catch all the exceptions

Reply:
We would get error events, isn't that enough ?

Reply:
`node_startup` catches all the exceptions and ignores it (adds to queue, but anyway later is ignored)

Reply:
`verify_node_startup` checks the exceptions caught during the `node_startup`. So, not ignored.